django/core/checks/security/base.py

```diff
@@ -68,8 +68,9 @@
 
 W009 = Warning(
     "Your SECRET_KEY has less than %(min_length)s characters or less than "
-    "%(min_unique_chars)s unique characters. Please generate a long and random "
-    "SECRET_KEY, otherwise many of Django's security-critical features will be "
+    "%(min_unique_chars)s unique characters, or it's django-generated secret key starting with dj::insecure."
+    "Please generate a long and random SECRET_KEY, "
```

Comment:
Maybe `"Please generate a sufficiently long and random SECRET_KEY"`? Or should we maybe repeat `min_unique_chars` here to be clear?

Reply:
I don't think we should repeat information from previous line, but agree with your version of sentence.